layouts/_default/baseof.html

```diff
@@ -13,7 +11,6 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="content-type" content="text/html; charset=utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="google-site-verification" content="8cyiY374WL2LGr87fBghcdJCRx6BaPNBvLjeZlk6tyc" />
```

Comment:
Whoops. I think I left this in on accident from when I ported my old site to a theme. That's not good 😵